camel/toolkits/hybrid_browser_toolkit/ws_wrapper.py

```
@@ -264,20 +275,44 @@ async def start(self):
+        tasks_to_cancel = [
+            ('_receive_task', self._receive_task),
+            ('_log_reader_task', self._log_reader_task),
+        ]
+
+        for _, task in tasks_to_cancel:
+            if task and not task.done():
+                task.cancel()
+                try:
+                    await task
+                except asyncio.CancelledError:
+                    pass
```

Ruff (0.12.2)

Use `contextlib.suppress(asyncio.CancelledError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(asyncio.CancelledError)`

(SIM105)